tests/Fixtures/inc/Engine/Optimization/CSSTraitApplyFontDisplaySwap.php

```diff
@@ -46,7 +46,7 @@
 CSS
 		],
 
-		'shouldNotChangeFontDisplayAttributeWhenAlreadySetInRule' => [
+		'shouldChangeFontDisplayAttributeWhenAlreadySetInRule' => [
```

Comment:
I think we need to add more test cases on combineV2, minify CSS, combine CSS.
To handle forcing font-display to be swap.

Reply:
@engahmeds3ed I've added a testcase for CombineV2 below. I don't think we can add for Minify/Combine CSS, as those testcases don't evaluate the contents of the results, rather that the expected files are created and combined -- both of these use the CSSTrait to perform the swap, and that is under test here in the `CSSTraitApplyFontDisplaySwap` fixture - so the code is covered.

Reply:
Ok great.